arbnode/seq_coordinator.go

```diff
@@ -215,6 +215,11 @@ func (c *SeqCoordinator) chosenOneUpdate(ctx context.Context, msgCountExpected,
 			return err
 		}
 		if remoteMsgCount > msgCountExpected {
+			if messageData == nil && c.CurrentlyChosen() {
+				// this was called from update(), while msgCount was changed by a call from SequencingMessage
+				// no need to do anything
+				return nil
```

Comment:
Supposing that this really was failing non-spuriously, wouldn't the lockout suddenly disappear and we'd have no idea why? Perhaps we should keep the Info log line below but just not return an error.

Reply:
No, this function failing doesn't do anything to the lockout.
The lockout was just extended by the competing SequencingMessage, so no risk of it timing out because we missed this update either.

Reply:
Fair enough. In theory another process could've updated the message count but since we check if we're chosen that'd indicate a much larger issue I suppose.